src/util/validate_model.cc

```diff
@@ -0,0 +1,100 @@
+#include "z3++.h"
+#include <ilang/util/log.h>
+#include <ilang/util/validate_model.h>
```

Comment:
Header include system should be 
1. key header (i.e., validate_model.h)
2. system libraries
3. local (e.g., log.h)

Reply:
Seems the clang-format will auto arrange header files' order.

Reply:
It will, and it should. But it only sorts within a section (separated by new line). Like this [example](https://github.com/Bo-Yuan-Huang/ILAng/blob/07c9c674875daa8be15368bb423159a18a3f7051/src/verilog-in/verilog_analysis.cc#L4).

Reply:
I see, thanks